### PR TITLE
Release 1.0.0.dev0

### PR DESCRIPTION
**Pull request by adriendelsalle**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by nicoddemus** [comment]
Hi @adriendelsalle,

Thanks for the PR, but the conda package should follow the PyPI releases, not create a new release by itself.

**Comment by nicoddemus** [comment]
Ahh sorry, didn't realize https://pypi.org/project/pluggy/1.0.0.dev0 is actually there (but hidden by default of course).

File: recipe/conda_build_config.yaml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+channel_targets:
```
Comment (nicoddemus) [inline recipe/conda_build_config.yaml:1]: Why is this needed?

Comment (adriendelsalle) [reply recipe/conda_build_config.yaml:1]: Hi @nicoddemus !
Thanks a lot for this review. This will publish the package with a `dev` label, making it not available for conda-forge as a stable release but available using the `dev` label!

Comment (nicoddemus) [reply recipe/conda_build_config.yaml:1]: Ahh right, found the related docs:

https://conda-forge.org/docs/maintainer/knowledge_base.html?highlight=channel_targets#creating-a-pre-release-build

Thanks!

**Review by nicoddemus — APPROVED** [review]

**Comment by adriendelsalle** [comment]
Thanks!
